commons/com.b2international.index.lucene/src/com/b2international/index/json/Index.java

```diff
@@ -199,7 +209,7 @@ private void addToDoc(final Document doc, final String name, final JsonNode node
 			final Field field = mapping.getField(name);
 			final Class<?> fieldType = NumericClassUtils.unwrapCollectionType(field);
 			
-			if (docValues && NumericClassUtils.isCollection(field)) {
+			if (docValues && NumericClassUtils.isCollection(fieldType)) {
```

Comment:
`fieldType` is now an eg. `int` if the field's type was `IntCollection` because of the unwrapping above, so this check will not throw for the cases that we expect it to.